internal/verifier/change_stream.go

```diff
@@ -232,17 +327,17 @@ func (verifier *Verifier) iterateChangeStream(
 		case <-ctx.Done():
 			return ctx.Err()
 
-		// If the changeStreamEnderChan has a message, the user has indicated that
-		// source writes are ended. This means we should exit rather than continue
+		// If the ChangeStreamEnderChan has a message, the user has indicated that
+		// source and destination writes are ended. This means we should exit rather than continue
```

Comment:
I don’t know that the “and destination” part is gainful here. It might suggest that user writes to the destination are expected during a migration, which they definitely aren’t.

Reply:
I think it means that the migration tool has stopped/committed. I'll clarify in this comment.